Tests/CacheAdvanceTests/CacheHeaderHandleTests.swift

```diff
@@ -49,7 +49,7 @@ final class CacheHeaderHandleTests: XCTestCase {
         XCTAssertEqual(headerHandle2.offsetInFileOfOldestMessage, 2000)
     }
 
-    func test_synchronizeHeaderData_returnsDefaultVersionWhenUnexpectedVersionIsOnDisk() throws {
+    func test_synchronizeHeaderData_throwsFileCorruptedWhenDefaultVersionWhenUnexpectedVersionIsOnDisk() throws {
```

Comment:
I think I mentioned this elsewhere but it could be nice for the error that is thrown to be more specific. Is the file really corrupted?

Reply:
yeah this shouldn't `throw`. Fixing.